tools/sqlite2txt/CMakeLists.txt

```diff
@@ -2,5 +2,5 @@ add_executable(sqlite2txt
         main.cpp
 )
 
-target_link_libraries(sqlite2txt SQLite::SQLite3)
+target_link_libraries(sqlite2txt SQLite::SQLite3 pthread dl)
```

Comment:
The missing symbols from pthread and dl are referenced by sqlite. There is no direct sqlite2txt's dependence on those.